src/NetMQ.Tests/EventDelegatorTests.cs

```diff
@@ -40,6 +38,7 @@ void DelegatorHandler(object sender, Args<double> args)
             delegator.Event += DelegatorHandler;
 
             Assert.NotNull(Source);
+            Assumes.NotNull(Source);
```

Comment:
xUnit won't be annotated until v3 is released, so there are a few suppressions following `Asset.NotNull` in tests unfortunately.

https://github.com/xunit/xunit/pull/2044